test/suite/test_tiered06.py

```diff
@@ -59,6 +59,7 @@ def conn_extensions(self, extlist):
         # S3 store is built as an optional loadable extension, not all test environments build S3.
         if self.ss_name == 's3_store':
             #config = '=(config=\"(verbose=1)\")'
+            # config = '=(config=\"(verbose=[api:1,version,tiered:-3])\")'
```

Comment:
Could you keep one of the commented out "config" and remove the other ones. Maybe merge them together and say ```tiered:1``` ?

Reply:
I've just changed it to keep `config = '=(config=\"(verbose=[api:1,version,tiered:1])\")'`